examples/utils/plot_surface2ind_topo.py

```diff
@@ -1,3 +1,10 @@
+"""
+Utils: surface2ind_topo
```

Comment:
irrespective of it we change the order, Magnetotellurics should be call Natural Source Electromagnetics